Rakefile.dist

```diff
@@ -65,6 +66,14 @@ task :install => :build do |t|
 
   # ssh
   mkdir_p "#{DATAROOTDIR}/foreman/.ssh"
+
+  # cron scripts
+  mkdir_p "#{SYSCONFDIR}/cron.hourly"
+  cp_p "script/passenger-recycler.rb", "#{SYSCONFDIR}/cron.hourly/passenger-recycler"
+
+  # configuration files
+  mkdir_p "#{SYSCONFDIR}/foreman"
+  cp_p "script/passenger-recycler.rb.conf.example", "#{SYSCONFDIR}/foreman/passenger-recycler.rb.conf"
```

Comment:
@domcleal any objections against starting  moving configuration files out of packaging repo into core repo? I believe `Rakefile.dist` is the correct place of this, less work in the `foreman-packaging`.